srcpkgs/ecl/template

```diff
@@ -0,0 +1,26 @@
+# Template file for 'ecl'
+pkgname=ecl
+version=16.1.3
+revision=1
+build_style=gnu-configure
```

Comment:
add this line directly below build_style

```
configure_args="--enable-gmp=system --enable-boehm=system --enable-libatomic=system"
```

Reply:
done.